be/src/exec/local_file_writer.cpp

```diff
@@ -43,7 +42,7 @@ Status LocalFileWriter::open() {
         if (success != 0) {
             std::stringstream ss;
             ss << "Seek to start_offset failed. offset=" << _start_offset << ", errno= " << errno
-               << ", description=" << get_str_err_msg();
+               << ", description=" << std::strerror(errno);
```

Comment:
Will operator<< change the value of errno? It will be safe to save errno in a temporary variable. I see the original function does it in this way

Reply:
no, iostream when throw exception when error happen.
```c++
// /usr/include/c++/12/bits/basic_ios.h
      void
      _M_setstate(iostate __state)
      {
	// 27.6.1.2.1 Common requirements.
	// Turn this on without causing an ios::failure to be thrown.
	_M_streambuf_state |= __state;
	if (this->exceptions() & __state)
	  __throw_exception_again;
      }
```